iggy-enrich-demo/iggy_enrich_demo.ipynb

```diff
@@ -0,0 +1,791 @@
+    "vacation_rental_columns = list(rental_data_clean.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c96d084e-2c27-4a6a-94e5-c47f54e8af0e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "features = [\n",
+    " 'acs_housing_units_built_2000_to_2009_cbg',\n",
+    " 'acs_housing_units_built_2010_to_2013_cbg',\n",
+    " 'acs_housing_units_built_2014_or_later_cbg',\n",
+    " 'acs_median_age_cbg',\n",
+    " 'acs_pct_households_cohabiting_couple_with_children_cbg',\n",
+    " 'acs_pct_households_female_head_with_children_cbg',\n",
+    " 'acs_pct_households_male_head_with_children_cbg',\n",
+    " 'acs_pct_households_married_couple_with_children_cbg',\n",
+    " 'acs_pct_housing_units_built_1939_or_earlier_cbg',\n",
+    " 'acs_pct_housing_units_built_1940_to_1949_cbg',\n",
+    " 'acs_pct_housing_units_built_1950_to_1959_cbg',\n",
+    " 'acs_pct_pop_commutes_by_public_transport_any_cbg',\n",
+    " 'park_pct_area_intersecting_boundary_qk_isochrone_walk_10m',\n",
+    " 'poi_count_cbg',\n",
+    " 'poi_count_qk_isochrone_walk_10m',\n",
+    " 'poi_count_zipcode',\n",
+    " 'poi_is_bar_count_per_sqkm_qk_isochrone_walk_10m',\n",
+    " 'poi_is_brand_crossfit_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_brand_trader_joes_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_convenience_store_or_pharmacy_count_per_sqkm_qk_isochrone_walk_10m',\n",
+    " 'poi_is_cultural_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_games_and_amusement_recreation_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_grocery_store_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_historical_site_count_per_sqkm_qk_isochrone_walk_10m',\n",
+    " 'poi_is_museum_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_nature_recreation_count_per_sqkm_qk_isochrone_walk_10m',\n",
+    " 'poi_is_parking_count_per_capita_qk_isochrone_walk_10m',\n",
+    " 'poi_is_performance_venue_count_qk_isochrone_walk_10m',\n",
+    " 'poi_is_restaurant_count_per_sqkm_qk_isochrone_walk_10m',\n",
+    " 'water_intersects_zipcode',\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d847211b-bfed-4bf4-8a7e-9b31adb745e1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pkg_spec = {\n",
+    "    \"iggy_version_id\": \"20211209155137\",\n",
+    "    \"crosswalk_prefix\": \"tx_austin_quadkeys\",\n",
+    "    \"base_loc\": \"austin-datasets/\",\n",
+    "    \"iggy_prefix\": \"tx_austin_quadkeys\"\n",
+    "}\n",
+    "pkg = LocalIggyDataPackage(**pkg_spec)\n",
+    "iggy = IggyEnrich(iggy_package=pkg)\n",
+    "\n",
+    "iggy.load(features=features)\n",
+    "rentals_enriched_df = iggy.enrich_df(rental_data_clean, latitude_col=\"latitude\", longitude_col=\"longitude\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59a1d7d7-7bd1-4cdb-b292-85d2af9b7511",
+   "metadata": {},
+   "source": [
+    "Now we have enriched the data, we will plot maps at the different boundary levels"
```

Comment:
...plot maps at various boundary levels, from coarse-grained (zip code) to fine-grained (quadkey zoom 19)